### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Oluwadamilareolusakin**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently under review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <link rel="stylesheet" href="sphinx/styles.css">
+    <link rel="stylesheet" type = "text/css" href="css/all.css">
+    <link rel="stylesheet" type = "text/css" href="css/index.css">
+</head>
+
+<body>
+  <main>
+    <div class="sign-up-holder column">
+  
+      <div class="col-ms-7 col-xs-24 column sign-up-left justify-content-center">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

`col-xs-24` is used but seems to not be defined in any of the CSS files. Is the Grid-based framework columns sized from 1/12th or 1/24th of the row to 100% of the row? :thinking:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            <i class = "fas fa-user"></i> Hear what people are talking about.
+          </div>
+          <div class="lin3">
+          <i class = "fab fa-twitter"></i> Join the conversation.
+          </div>
+        </div>
+      </div>
+         
+  
+      <div class="sign-up-right col-ms-7 col-xs-24 column align-items-center">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

`col-xs-24` is used but seems to not be defined in any of the CSS files. Is the Grid-based framework columns sized from 1/12th or 1/24th of the row to 100% of the row? :thinking:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            <button class="EdgeButton EdgeButton--medium2 EdgeButton--secondary">
+              Log in
+            </button>
+          </div>
+      
+        </div>
+      </div>
+    </div>
+
+    <div class="sign-up-middle column align-items-center col-xs-24 ">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

`col-xs-24` is used but seems to not be defined in any of the CSS files. Is the Grid-based framework columns sized from 1/12th or 1/24th of the row to 100% of the row? :thinking:

**Comment by Oghenebrume50** [comment]
reviewing........

**Pull request merged**